src/amo/components/CollectionDetails/index.js

```diff
@@ -70,30 +91,54 @@ export class CollectionDetailsBase extends React.Component<InternalProps> {
           ]}
         />
         {collection &&
-          (editing || showEditButton) && (
+          showEditButton &&
+          !editing && (
+            <Button
+              buttonType="neutral"
+              className="CollectionDetails-edit-button"
+              puffy
+              to={{
+                pathname: collectionEditUrl({ collection }),
+                query: convertFiltersToQueryParams(filters),
+              }}
+            >
+              {i18n.gettext('Edit this collection')}
```

Comment:
Well, it takes you into "Edit" mode which allows you to edit the add-ons, but also edit the details by thereafter clicking the "Edit collection details" button. I'm not sure it's 100% accurate to change it to say "Edit the add-ons in this collection" because if you also want to edit the details you first have to click this button.

@MeridelW what do you think?

Reply:
ah, that sort of makes sense. I did not think that way.